tsfm_public/resources/model_paths_config/ttm.yaml

```diff
@@ -0,0 +1,46 @@
+ibm-granite:
+    r1:
+        model_card: "ibm-granite/granite-timeseries-ttm-r1"
+        revision:
+            512: 
```

Comment:
It is not clear what the two nested levels indicate. Revision could be a simple list of maps, each map defines:
context_length, prediction_length, model_revision.

Reply:
Simplified the yaml.